kubernetes/helm/pinot-github-events-setup.yml

```diff
@@ -0,0 +1,280 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: examples
+  namespace: pinot-quickstart
+data:
+  pullRequestMergedEvents_realtime_table_config.json: |-
+    {
+      "tableName": "pullRequestMergedEvents",
```

Comment:
pull_request_merge_event

Reply:
Will keep this for now, otherwise it'll be difficult to make any more changes to any current setup